rpc_jobs/multi_node_aio.yml

```diff
@@ -3,16 +3,20 @@
     series:
       - mitaka:
           branch: mitaka-13.1
-          TEST_SET: "scenario defcore api heat_api smoke"
+          USER_VARS:
```

Comment:
The multi-node AIO jobs don't have the USER_VARS as a parameter currently like the AIO jobs do. You'll want to copy https://github.com/rcbops/rpc-gating/blob/master/rpc_jobs/params.yml#L81-L84 and add it to the parameters.